src/components/common/Tab/TabItem.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import { ReactNode } from 'react';
+
+import { useTab } from './TabGroup';
+
+import cn from '@utils/cn';
+
+export interface TabItemProps {
+  value: string;
+  children: ReactNode;
+}
+
+export function TabItem({ value, children }: TabItemProps) {
+  const { activeValue, setActiveValue } = useTab();
+  const isActive = value === activeValue;
+
+  return (
+    <li
+      className={cn(
+        'header-18 text-gray-900 w-[72px] h-[32px] flex justify-center items-center border-b-[4px]',
+        isActive ? ' border-primary-500' : 'border-transparent',
+      )}
```

Comment:
cn이 제공하는 기능을 사용하기 위해서 아래처럼 작성해보면 어떨까요?
제가 지금 코드를 실행해볼수가 없어서... 아래처럼 수정하면 잘 되는지도 궁금한데 한번 확인해주시면 감사하겠습니다 ㅋㅋㅋㅋㅋ

```suggestion
      className={cn(
        'header-18 text-gray-900 w-[72px] h-[32px] flex justify-center items-center border-b-[4px] border-transparent',
{ 'border-primary-500' : isActive }
      )}
```

Reply:
옹 잘됩니다! 이렇게 사용해야겠네요🥹